phenogenon/helper.py

```diff
@@ -87,7 +85,7 @@ def _parse_config():
             result[section][option] = value
     return result
 
-
+CONFIG_FILE = 'configure.cfg'
```

Comment:
is `configure.cfg` hardcode here?

Reply:
yes that's the default value - it was previously hard-coded in the `_parse_config()` method above...

This way, I can change the value in "./scripts/phenogeneon" by running helper.CONFIG_FILE = 'other_file.cfg'

Not sure if this is the best way to do this?